packages/contracts-bedrock/test/properties/medusa/SuperchainERC20Factory.t.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.25;
+import {Test} from "forge-std/Test.sol";
+
+import { SuperchainERC20Factory } from "src/L2/SuperchainERC20Factory.sol";
+
+contract SuperchainERC20FactoryFuzz is Test{
+    struct DeployParams {
+        address remoteToken;
+        string name;
+        string symbol;
+        uint8 decimals;
+    }
+
+    SuperchainERC20Factory internal factory;
+
+    constructor() {
+        factory = new SuperchainERC20Factory();
+    }
+
+    // this is a stateless check, so halmos will probably supersede it
+    function testContractAddressDependsOnParams(DeployParams memory left, DeployParams memory right) external {
```

Comment:
on the opUSDC campaign we used this standard for test contracts names:
```suggestion
    function fuzz_contractAddressDependsOnParams(DeployParams memory left, DeployParams memory right) external {
```

I think we should stick to that unless is a blocker for medusa